package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "dnd-beyond-dice-chrome-extension",
+    "version": "1.0.0",
+    "description": "A Chrome Extension that listens for dice rolls on D&D Beyond Character Sheets (e.g. [Remigius](https://www.dndbeyond.com/profile/kamnikite/characters/23934295)) and posts the results into a Discord server via a bot.",
+    "main": "content.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/mikemfleming/dnd-beyond-dice-chrome-extension.git"
+    },
+    "author": "",
+    "license": "ISC",
```

I think this should be `MIT` since we're using the MIT license in [`LICENSE`](https://github.com/mazzola/dnd-beyond-dice-chrome-extension/blob/master/LICENSE)

```suggestion
    "license": "MIT",
```